docs/user-guide/multiple_sources.md

```diff
@@ -0,0 +1,62 @@
+# Multiple Sources for an Application
```

Comment:
Could you add a section to this doc showing how to use multi-sources to apply a different values.yaml to a Helm chart?